packages/datadog-plugin-couchbase/src/index.js

```diff
@@ -32,8 +32,9 @@ class CouchBasePlugin extends Plugin {
       tags
     })
 
-    if (bucket) span.setTag('couchbase.bucket.name', bucket.name) // semver >=2 <3
-    if (collection) span.setTag('couchbase.collection.name', collection.name) // semver >=3
+    Object.keys(containers).forEach(containerType => {
```

Comment:
Similar to the above, let's keep both bucket and collection explicitly, but without the comments since buckets were not removed in later versions.